test/evergreen.yml

```diff
@@ -3430,7 +3429,183 @@ tasks:
+            
             ${test_env_vars|} test/unittest/unittests
+
+            ${test_env_vars|} $CTEST -R '(ex_|lsm_workload|filesys|metadata_salvage|col_efficiency|dup_index_collator|compact_quick|test_checkpoint_4_mixed_timestamps|test_checkpoint_6_row_named_prepare|random_directio|scope|join_main_table_row|pad_byte_collator|meta_ckptlist_get_alloc)'
+
+            ${test_env_vars|} /opt/mongodbtoolchain/v4/bin/python3 ../test/suite/run.py test_rollback_to_stable03
+            ${test_env_vars|} /opt/mongodbtoolchain/v4/bin/python3 ../test/suite/run.py lsm
+            ${test_env_vars|} /opt/mongodbtoolchain/v4/bin/python3 ../test/suite/run.py test_prepare01
+            ${test_env_vars|} /opt/mongodbtoolchain/v4/bin/python3 ../test/suite/run.py -p test_prepare02
+            ${test_env_vars|} /opt/mongodbtoolchain/v4/bin/python3 ../test/suite/run.py test_prepare03
+            ${test_env_vars|} /opt/mongodbtoolchain/v4/bin/python3 ../test/suite/run.py test_prepare04
+            ${test_env_vars|} /opt/mongodbtoolchain/v4/bin/python3 ../test/suite/run.py test_truncate02
+            ${test_env_vars|} /opt/mongodbtoolchain/v4/bin/python3 ../test/suite/run.py test_tiered04
+            ${test_env_vars|} /opt/mongodbtoolchain/v4/bin/python3 ../test/suite/run.py test_sweep01
+            ${test_env_vars|} /opt/mongodbtoolchain/v4/bin/python3 ../test/suite/run.py test_import01
+            ${test_env_vars|} /opt/mongodbtoolchain/v4/bin/python3 ../test/suite/run.py flcs
+            ${test_env_vars|} /opt/mongodbtoolchain/v4/bin/python3 ../test/suite/run.py test_backup13
+            ${test_env_vars|} /opt/mongodbtoolchain/v4/bin/python3 ../test/suite/run.py test_backup24
+            ${test_env_vars|} test/cppsuite/run -t bounded_cursor_stress -f test/cppsuite/configs/bounded_cursor_stress_default.txt
+            ${test_env_vars|} test/cppsuite/run -t cache_resize -f test/cppsuite/configs/cache_resize_default.txt
+            ${test_env_vars|} test/cppsuite/run -t hs_cleanup -f test/cppsuite/configs/hs_cleanup_default.txt
+            ${test_env_vars|} ./test/csuite/wt1965_col_efficiency/test_wt1965_col_efficiency
+            ${test_env_vars|} ./test/csuite/wt2999_join_extractor/test_wt2999_join_extractor
+            ${test_env_vars|} ./test/csuite/wt3135_search_near_collator/test_wt3135_search_near_collator
+            ${test_env_vars|} ./test/csuite/wt3184_dup_index_collator/test_wt3184_dup_index_collator
+            ${test_env_vars|} ./test/csuite/wt4156_metadata_salvage/test_wt4156_metadata_salvage
+            ${test_env_vars|} ./test/csuite/wt2403_lsm_workload/test_wt2403_lsm_workload
+
+            ${test_env_vars|} WIREDTIGER_HOME=ex1 examples/c/ex_all/ex_all & ${test_env_vars|} WIREDTIGER_HOME=ex2 examples/c/ex_schema/ex_schema
+            # Rest of the examples
+            cd examples/c/
+            ${test_env_vars|} $CTEST --output-on-failure 2>&1
+            cd ../../
+
+            # Faster csuite tests. 30% in ~1 minute
```

Comment:
Does "30%" indicate the code coverage % the below set of tests is able to achieve?

Reply:
That comment was copy/pasted from the original competition entry, and is now obsolete.